apis/base.es6.js

```diff
@@ -25,17 +25,6 @@ class BaseAPI {
     this.config = base.config;
     this.cache = base.cache;
     this.event = base.event;
-
-    if (base.config) {
-      this.origin = base.config.origin;
-
-      if (base.config.origins) {
```

Comment:
:no_bell: It doesn't look like we use origins. It looks like its just here to share configs between multiple api instances? Removing it seems simpler, just curious if it needs to be preserved. If it does `origin` could be a `getter`